### ISSUE TITLE
Writeback_cache support

### ISSUE THREAD

**Issue by DaRacci** [mpartel/bindfs#157]
Looking into fuse 3, there is supposed to be a good performance improvement from using the `writeback_cache` mode is this something that can already be passed through to the fuse options in some way?

I've tried with `-o writeback_cache` but it wasn't recognised.

**Comment by mpartel** [comment]
No, supporting it correctly requires some [small code changes](https://github.com/libfuse/libfuse/blob/682e87dbc0a18a703fd4a746b5ef7ee22fb559cf/include/fuse.h#L442-L455). That's probably why libfuse doesn't make the option available by default.
